lib/quickcrawl.js

```diff
@@ -63,7 +63,7 @@ module.exports = function crawl(url, successCallback, failCallback) {
         throw new Error("Can't crawl with unspecified path.");
     }
 
-    var tmpCrawler = new Crawler(url.hostname(), url.path(), url.port() || (url.protocol() === "https" ? 443 : 80));
+    var tmpCrawler = new Crawler(url.href());
```

Comment:
I'm wondering whether we should just pass uri through, and do the validation in `crawler.js`.